crates/pixi_consts/src/consts.rs

```diff
@@ -47,6 +46,13 @@ lazy_static! {
     pub static ref DEFAULT_PYPI_INDEX_URL: Url = Url::parse("https://pypi.org/simple").unwrap();
 }
 
+#[macro_export]
+macro_rules! pixi_version {
+    () => {
+        option_env!("PIXI_VERSION").unwrap_or(env!("CARGO_PKG_VERSION"))
```

Comment:
The cargo pkg version here doesnt quite work because this is used in a different crate than the pixi executable itself.

So far we have hardcoded this and used an automation to update this.